### DISCUSSION TITLE
Is there a way to connect existing kafka to this operator

### DISCUSSION THREAD

**Discussion by EsDmitrii** [strimzi/strimzi-kafka-operator#9611] [Q&A]
Is there any way to connect an existing kafka deployed on kubernetes or on a VM to this operator to manage topics, etc through the strimzi operator?

**Comment by scholzj** [comment]
You can use the _standalone_ topic operator to manage your Kafka topics assuming you use some supported authentication mechanisms etc. There are installation files on GitHub and little bit of it is covered in the docs. The rest might be a bit up to you to figure it out.

**Reply by EsDmitrii** [reply]
Can you share with me the link to standalone topic operator and to docs, please? Can't find anything here
Thank you in advance!

**Reply by scholzj** [reply]
I do not have the links at hand right now. But if you search for _standalone topic operator_ in the docs, you should be able to find it. On GitHub it is in the installation files.
